### DISCUSSION TITLE
Guidance with the API

### DISCUSSION THREAD

**Discussion by StoyanDimitrov** [WeblateOrg/weblate#7487] [Q&A] [answered]
I'm trying to build an automatic dashboard of the projects where I contribute. I've partial success achieving this with the API, meaning I have a list of project, component tuples. Currently I manually update this list but would like to fetch that data with the help of the API. What I'm struggling is to find a way to fetch only the projects where I contribute.

**Comment by nijel** [comment, marked as answer]
I don't think there is currently a good API call for that. You can browse your recent changes, but that won't cover projects you've translated in the past.
